packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -107,6 +113,20 @@ module.exports = (
           }).then(() => {
             const markdownAST = remark.parse(markdownNode.internal.content)
 
+            if (pathPrefix) {
```

Comment:
Oh one thing I just remembered — `pathPrefix` is always defined. It just is set to an empty string if you're not building with `--prefix-paths` https://github.com/gatsbyjs/gatsby/blob/41ff67db854899c1bc8304654460a27330b0537f/packages/gatsby/src/utils/api-runner-node.js#L61

So this if isn't necessary.

Reply:
But if `pathPrefix` is defined and empty this condition will skip unneeded AST traverse

Reply:
Oh right :-) Forgot empty strings are falsey not truthy — 🙄  @ JS